LanguageFeatures/Patterns/pattern_uses_A02_t02.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion
+/// The static type of a switch expression is the least upper bound of the
+/// static types of all of the case expressions.
+///
+/// @description Check that it is a compile-time error to assign a switch
+/// expression to a type which is not the least upper bound of the static types
+/// of all of it's case expressions.
```

Comment:
```suggestion
/// of all of its case expressions.
```
("its" is a special exception, there's no apostrophe in the possessive form)

Reply:
I think this one slipped under the carpet: The above text means "of all of it is case expressions", which is probably not intended.